quiz/web/tailwind.config.cjs

```diff
@@ -1,8 +1,16 @@
 /** @type {import('tailwindcss').Config} */
+
 module.exports = {
     content: ["./index.html", "./src/**/*.{js,ts,jsx,tsx}"],
     theme: {
+        colors: {
+            primary: "#4459a9",
```

Comment:
it's not used anywhere

Reply:
Robert has a whole palette of colors that you have to enter here to use it. (Because there are no such colors in the MUI palette) I did it at the beginning. Due to Tailwind failing to overwrite the MUI styles, it is not used. In the future, in my opinion, this should be done so as not to repeat the code. I wanted to hear your opinion on this, so I left it.